includes/Post/HandleFinalDataOfCreatePost.php

```diff
@@ -159,7 +159,26 @@
 		echo "<br><br><h3 style = 'color:white ;'> $post Post  already exists .</h3>";
 		die ();
 	}
-
+	
+	// create table for asigning posts
+	$stmt = $sqlConn->prepare("CREATE TABLE IF NOT EXISTS $assignPostTable ($postTitle VARCHAR(100) NOT NULL, $assignedUser VARCHAR(100), INDEX idx USING BTREE ($postTitle))");
+	if ( ! $stmt->execute() )
+	{
+		echo"there is a problem with database<br>";
+		die ();
+	}
+	
+	$stmt->close();
+	
+	// asign post ''
+	$stmt2 = $sqlConn->prepare("INSERT INTO $assignPostTable VALUES (?,\"\")");
+	$stmt2->bind_param ('s', $post );
+	if ( ! $stmt2->execute() )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line indented incorrectly; expected 0 spaces, found 4
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 5

<!-- Comment by [SideCI](https://sideci.com) -->